calm/samples/traderx/controls/flow-sla-control-requirement.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/traderx/controls/flow-sla-control-requirement.json",
+  "title": "Flow SLA Control",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://calm.finos.org/draft/2024-10/meta/control-requirement.json"
+    }
+  ],
+  "properties": {
+    "control-id": {
+      "const": "flow-sla-001"
+    },
+    "name": {
+      "const": "Flow SLA Control"
+    },
+    "description": {
+      "const": "Ensure that each flow meets the defined Service Level Agreement (SLA)"
+    },
+    "latency": {
+      "type": "object",
+      "properties": {
+        "expected-latency": {
+          "$ref": "https://calm.finos.org/draft/2024-10/meta/units.json#/defs/time-unit",
+          "description": "The expected latency for completion of flow"
+        },
+        "alert-threshold": {
+          "$ref": "https://calm.finos.org/draft/2024-10/meta/units.json#/defs/time-unit",
+          "description": "If latency exceeds this threshold, an alert will be triggered."
+        }
+      },
+      "required": [
+        "expected-latency",
+        "alert-threshold"
+      ]
+    },
+    "throughput": {
```

Comment:
Doesn't throughput also require a number of messages for the time period? Isn't the time-unit reference just going to give us the period it's recorded over?

Reply:
I was thinking about this last night and had the same comment for myself. I would say perhaps an enhancement to the timeunit spec to maybe contain something that represents whether dealing with the below distinctions. 

[amount] **per** unit of time e.g. 5 per second
**every** [amount] unit of time e.g. every 5 seconds

Reply:
I would say unit of time as a standalone as it is makes sense, but we should have separate definitions which use the time unit combined with the amount as a separate definition.

Reply:
As discussed offline, we'll look to make throughput a clearer type - https://github.com/finos/architecture-as-code/issues/466